src/components/MessageList/VirtualizedMessageList.tsx

```diff
@@ -159,6 +174,18 @@ const VirtualizedMessageListWithContext = <
     TypingIndicator = null,
   } = props;
 
+  const { userID } = client;
+  const lastRead = useMemo(() => channel.lastRead(), [channel]);
```

Comment:
```suggestion
  const lastRead = useMemo(() => channel?.lastRead(), [channel]);
```

can't we do this to handle channel being a partial type?